src/note.rs

```diff
@@ -2,11 +2,151 @@
+    pub const Cs1m: Self = Self::new(0 * 12 + 1);
+    pub const D1m: Self = Self::new(0 * 12 + 2);
+    pub const Ds1m: Self = Self::new(0 * 12 + 3);
+    pub const E1m: Self = Self::new(0 * 12 + 4);
+    pub const F1m: Self = Self::new(0 * 12 + 5);
+    pub const Fs1m: Self = Self::new(0 * 12 + 6);
+    pub const G1m: Self = Self::new(0 * 12 + 7);
+    pub const Gs1m: Self = Self::new(0 * 12 + 8);
+    pub const A1m: Self = Self::new(0 * 12 + 9);
+    pub const As1m: Self = Self::new(0 * 12 + 10);
+    pub const B1m: Self = Self::new(0 * 12 + 11);
+
+    pub const C0: Self = Self::new(1 * 12 + 0);
+    pub const Cs0: Self = Self::new(1 * 12 + 1);
+    pub const D0: Self = Self::new(1 * 12 + 2);
+    pub const Ds0: Self = Self::new(1 * 12 + 3);
+    pub const E0: Self = Self::new(1 * 12 + 4);
+    pub const F0: Self = Self::new(1 * 12 + 5);
+    pub const Fs0: Self = Self::new(1 * 12 + 6);
+    pub const G0: Self = Self::new(1 * 12 + 7);
+    pub const Gs0: Self = Self::new(1 * 12 + 8);
+    pub const A0: Self = Self::new(1 * 12 + 9);
+    pub const As0: Self = Self::new(1 * 12 + 10);
+    pub const B0: Self = Self::new(1 * 12 + 11);
+
+    pub const C1: Self = Self::new(2 * 12 + 0);
+    pub const Cs1: Self = Self::new(2 * 12 + 1);
+    pub const D1: Self = Self::new(2 * 12 + 2);
+    pub const Ds1: Self = Self::new(2 * 12 + 3);
+    pub const E1: Self = Self::new(2 * 12 + 4);
+    pub const F1: Self = Self::new(2 * 12 + 5);
+    pub const Fs1: Self = Self::new(2 * 12 + 6);
+    pub const G1: Self = Self::new(2 * 12 + 7);
+    pub const Gs1: Self = Self::new(2 * 12 + 8);
+    pub const A1: Self = Self::new(2 * 12 + 9);
+    pub const As1: Self = Self::new(2 * 12 + 10);
+    pub const B1: Self = Self::new(2 * 12 + 11);
+
+    pub const C2: Self = Self::new(3 * 12 + 0);
+    pub const Cs2: Self = Self::new(3 * 12 + 1);
+    pub const D2: Self = Self::new(3 * 12 + 2);
+    pub const Ds2: Self = Self::new(3 * 12 + 3);
+    pub const E2: Self = Self::new(3 * 12 + 4);
+    pub const F2: Self = Self::new(3 * 12 + 5);
+    pub const Fs2: Self = Self::new(3 * 12 + 6);
+    pub const G2: Self = Self::new(3 * 12 + 7);
+    pub const Gs2: Self = Self::new(3 * 12 + 8);
+    pub const A2: Self = Self::new(3 * 12 + 9);
+    pub const As2: Self = Self::new(3 * 12 + 10);
+    pub const B2: Self = Self::new(3 * 12 + 11);
+
+    pub const C3: Self = Self::new(4 * 12 + 0);
+    pub const Cs3: Self = Self::new(4 * 12 + 1);
+    pub const D3: Self = Self::new(4 * 12 + 2);
+    pub const Ds3: Self = Self::new(4 * 12 + 3);
+    pub const E3: Self = Self::new(4 * 12 + 4);
+    pub const F3: Self = Self::new(4 * 12 + 5);
+    pub const Fs3: Self = Self::new(4 * 12 + 6);
+    pub const G3: Self = Self::new(4 * 12 + 7);
+    pub const Gs3: Self = Self::new(4 * 12 + 8);
+    pub const A3: Self = Self::new(4 * 12 + 9);
+    pub const As3: Self = Self::new(4 * 12 + 10);
+    pub const B3: Self = Self::new(4 * 12 + 11);
+
+    pub const C4: Self = Self::new(5 * 12 + 0);
```

Comment:
I think middle C = C3 = 60 is more common, this is the Yamaha standard where middle C = C4 = 60.
I'm ok with both but maybe we need to document this or something

Reply:
ah okay, i'm okay with c3 = 60 if it is more common, i'll fix it up

Reply:
Lets check with the usbd guys first, they were using the Yamaha convention too, they might have a good reason for that. (maybe it was standardized in the usb midi spec?)

Also, as this isnt really standardized. Whatever we choose, we should document it somewhere. But I can do that outside of this PR.